OasisDEX/oasis-borrow
# Gf/basic buy form info

# [Add info section for basic buy & basic sell](https://app.shortcut.com/oazo-apps/story/3872/basic-sell-changes-information)

Covers both buy and sell info tickets.
- 3872
- 3871
  
## Changes 👷‍♀️
- Added a new reusable info section component
- added a new item component to render contents of a list of items
  
## How to test 🧪
Go to OpenVaultView.tsx and update the current sidebar there to the following. Modify the items array below to change the content that appears in the info section.

```
<SellInfoSection
            colRatioAfterSell={'200'}
            multipleAfterSell={'2'}
            ratiotoPeformSell={'250'}
            nextSellPrice={{
              value: '1,295.00',
              dropDownValues: [
                {
                  label: 'price',
                  value: '$1,250.00',
                },
                {
                  label: 'price',
                  value: '$1,350.00',
                }
              ]
            }}
            setupTransactionCost={'150.00'}
            slippageLimit={'0.6'}
            collatAfterNextSell={{
              value: '11.2345',
              secondaryValue: '345,234'
            }}
            outstandingDebtAfterSell={{
              value: '657,890',
              secondaryValue: '567,909'
            }}
            ethToBeSoldAtNextSell={'12.32'}
            estimatedTransactionCost={{
              value: '$250',
              dropDownValues: [
                {
                  label: 'price',
                  value: '$123.00'
                }
              ]
            }}
          />

```
